app/views/users/index.html.erb

```diff
@@ -1,15 +1,19 @@
-<h1>Users</h1>
+<section>
 
-<p>Here is the list of users:</p>
+<div class="users-header">
+  <h2>Users list</h2>
+</div>
 
-<ul>
+<ul class="users-ul">
   <% @users.each do |user| %>
-  <li>
-    <div>User Image</div>
-    <div>
+  <li class="users-li">
+    <div><%= image_tag user.photo %></div>
+    <div class="user-info">
       <h2><a><%= user.name %></a></h2>
-      <span>Number of posts:<%= user.posts_counter %></span>
+      <p>Number of posts:<%= user.posts_counter %></p>
     </div>
   </li>
 <% end %>
 </ul>
```

Comment:
- [ ] As required for the project, kindly make sure your design sticks as much as possible to the design of the [wireframe needed](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_root_users.png)